public/css/style.css

```diff
@@ -18,11 +27,74 @@ main {
     padding: 5px 10px;
 }
 
+h1 {
+    color: white;
+}
+
 a {
     text-decoration: none;
 }
 
+nav a {
+    color: white;
+    padding-left: 30px;
+    padding-right: 30px;
+}
+
+nav a:hover {
```

Comment:
black background looks not so pretty and better do not change the position of other elements when hover over a certain element